kerchunk/kerchunk-in-practice.ipynb

```diff
@@ -0,0 +1,2731 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    fs_read = fsspec.filesystem("s3", anon=True, skip_instance_cache=False)
</pre><p>It might be useful to mention the param usage. </p><p> </p><ul><li>We set <code>anon=True</code> because this specific dataset on AWS does not require users to be logged in to access.</li></ul><p> </p><p>(I don't know what <code>skip_instance_cache</code> does) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/cloud-optimized-geospatial-formats-guide/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>